Makefile

```diff
@@ -28,8 +28,11 @@ test:
 ci:
 	uv run pytest --cov techcity --migrations
 
+# Build the static site
 build:
-	docker compose build
+	@uv run manage.py distill-local out --force --collectstatic
+
+# Serve the static site locally
+serve-static:
+	@uv run python -m http.server --directory out
```

Comment:
you don't need to do this to develop locally,
can use normal `runserver` ; but using this to validate the build